cmd/notification/rm/rm.go

```diff
@@ -24,32 +26,36 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// NewCommand returns the rm command of type cobra.Command
-var byAge bool
+var age string
+var slug string
 
-//TODO need revisit ! What are the supported flags ? Are they consistent with the other commands ?
 func removeNotificationHandler(cmd *cobra.Command, args []string) (err error){
-	// Checking for args
-	if len(args) == 0 {
-		return errors.New("no notification ID/Name provided")
-	}
-	url := config.Conf.Clients["Notification"].Url() + clients.ApiNotificationRoute
-	if byAge {
-		url += "/age/" + args[0]
-	} else {
-		url += "/slug/" + args[0]
+	if age == "" && slug == "" {
+		return errors.New("at least one flag should be provided")
+	} else if age != "" && slug != ""{
+		return errors.New("age or slug flag should be provided, not both")
 	}
-	return request.DeletePrt(url, args[0])
+	url, deletedBy := constructUrl()
+	return request.DeletePrt(url, deletedBy)
 }
 
+// NewCommand returns the rm command of type cobra.Command
 func NewCommand() *cobra.Command {
 	var cmd = &cobra.Command{
-		Use:   "rm [notification slug or age]",
+		Use:   "rm [--slug | --age]",
 		Short: "Removes notification by slug or age",
-		Long:  `Removes a notification given its slug or age timestamp.`,
-		Args:  cobra.ExactValidArgs(1),
+		Long:  "Removes notifications by slug or age (the current notifications timestamp minus their last modification timestamp should be less than the age parameter)",
```

Comment:
s/less/greater ??
Say age is 5 hours, anything greater than 5 hours, like 5 hrs and 1 minute, 6 hrs, 24 hrs should get deleted? With notifications are we looking at when we it was created and current time?

Reply:
I took the description out of the api v1 documentation. 
The logic implemented is: 
curr_time - 5 hours  =  time in the past (5 hours earlier) (all in ms) 
If this past_time is less than the last modification time - the notification will be deleted  

But said in this way it should be really greater 
> the current notifications timestamp minus their last modification timestamp should be less than the age parameter